tests/amountClaim/pages/submitted.js

```diff
@@ -17,5 +19,14 @@ module.exports = {
 
   open () {
     I.amOnPage('/claim/submitted')
+  },
+
+  verifyTextInSubmittedPage () {
+    I.see('Your claim has been issued')
+    I.see('Claim number: 000LR001')
```

Comment:
this can only be run once in a container, fine for jenkins, not so good for local development
Maybe check if it contains LR? 
or don't check this

Reply:
Agreed, for time being i am going to remove it